site/content/release-notes/4.0.0.md

```diff
@@ -2,10 +2,242 @@
 title: "4.0.0"
 toc: true
 ---
-RDF4J 4.0.0 is a patch release that fixes 112 issues.
+RDF4J 4.0.0 is a major release of the Eclipse RDF4J framework. Some highlights:
+
+- Update to Java 11 as the minimally-required version of Java
+- Various improvements in the SparqlBuilder
+- Various speed and optimization improvements in the SPARQL query engine
+- Performance improvements in the SHACL engine
+- Performance improvements in the Memory Sail
+- Improved support for working with RDF4J in Spring and Spring Boot applications (see [documentation](/documentation/programming/spring) for details)
+- SeRQL and SPIN are no longer supported
+- Removal of several long-deprecated packages and interfaces
+- ...
```

Comment:
I think we can remove this line.

Reply:
When should 4.0.0 be releases? It would be best if the thread bug for LMDB is fixed first.

Reply:
It's pushed to maven already. Only thing left is to make the announcement on our site.

Reply:
But we can push 4.0.1 within just a few days if we want.